fetch.bs

```diff
@@ -6784,6 +6784,7 @@ dictionary RequestInit {
   DOMString integrity;
   boolean keepalive;
   AbortSignal? signal;
+  RequestDuplex? duplex;
```

Comment:
There's no need for this to be nullable.

Reply:
Done.